en/micro-integrator/docs/references/connectors/kafka-connector/kafka-inbound-endpoint-example.md

```diff
@@ -0,0 +1,88 @@
+# Kafka Inbound Endpoint Example
+
+The Kafka inbound endpoint of WSO2 ESB acts as a message consumer. It creates a connection to ZooKeeper and requests messages for either a topic/s or topic filters.
+
+## What you'll build
+This sample demonstrates how one way message bridging from Kafka to HTTP can be done using the inbound kafka endpoint.
+
+See [Configuring kafka connector](kafka-connector-configuration.md) for more information.
+
+## Configure the connector in WSO2 Integration Studio
+
+Follow these steps to set up the ESB Solution Project and the Connector Exporter Project.
+
+{!references/connectors/importing-connector-to-integration-studio.md!}
+
+1. Our project would look similar to the following (source view).
+
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<inboundEndpoint name="KAFKAListenerEP" sequence="kafka_process_seq" onError="fault" class="org.wso2.carbon.inbound.kafka.KafkaMessageConsumer" suspend="false" 
+    xmlns="http://ws.apache.org/ns/synapse">
+    <parameters>
+        <parameter name="sequential">true</parameter>
+        <parameter name="interval">10</parameter>
+        <parameter name="coordination">true</parameter>
+        <parameter name="inbound.behavior">polling</parameter>
+        <parameter name="value.deserializer">org.apache.kafka.common.serialization.StringDeserializer</parameter>
+        <parameter name="topic.names">test</parameter>
+        <parameter name="poll.timeout">100</parameter>
+        <parameter name="bootstrap.servers">localhost:9092</parameter>
+        <parameter name="group.id">hello</parameter>
+        <parameter name="contentType">application/json</parameter>
+        <parameter name="key.deserializer">org.apache.kafka.common.serialization.StringDeserializer</parameter>
+    </parameters>
+</inboundEndpoint>
+```
+Sequence
+
+```
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<sequence xmlns="http://ws.apache.org/ns/synapse" name="kafka_process_seq">
+   <log level="full"/>
+   <log level="custom">
+      <property xmlns:ns="http://org.apache.synapse/xsd" name="partitionNo" expression="get-property('partitionNo')"/>
+   </log>
+   <log level="custom">
+      <property xmlns:ns="http://org.apache.synapse/xsd" name="messageValue" expression="get-property('messageValue')"/>
+   </log>
+   <log level="custom">
+      <property xmlns:ns="http://org.apache.synapse/xsd" name="offset" expression="get-property('offset')"/>
+   </log>
+</sequence>
+```
+2. Right-click on the Composite Application Project and click on **Export Project Artifacts and Run**. Select **Run on Micro Integrator**.
+3. Micro Integrator will be started and the composite application will be deployed. You can further refer to the application deployed through the CLI tool. Make sure you first export the PATH as below.
+
+    ```
+    $ export PATH=/path/to/mi/cli/directory/bin:$PATH\
+    ```
+{! /references/connectors/exporting-artifacts.md !}
```

Comment:
Format is wrong. No slash and space before "references".

Reply:
Updated.